README.md

```diff
@@ -490,6 +490,85 @@ To quickly copy the value of a variable into the clipboard, use the keyboard pre
+
+For an Emacs Lisp Prelude file, it can run any code found in the `.el` file. (Even to dynamically call and get a temporary Bearer token!) For this reason, a yes-no warning prompt is presented when loading those files unless `verb-suppress-load-unsecure-prelude-warning` is set to non-nil value. This, of course, is another reason to GPG encrypt the configuration files to help prevent configuration poisoning. Here's an example of loading `verb-var`'s using an Emacs Lisp file:
+
+``` elisp
+; prod-foobar-env
+(message "Setting up verb variables!")
+; "email" variable is set from emacs user-mail-address
+(verb-set-var "email" user-mail-address)
+(verb-set-var "global_api_key" "hc33Vzco7TAkMKLNzIVgps7KiKLnUYIWJ7y9T")
+(verb-set-var "account_id" "Kb9SVZXtTVFYzt7rvgYv5WXJS31lh7OT")
+(verb-set-var "zone-ids" '(:example.com "IRMTk8T0RKgFxL3bL8KWv5FDDVtoe1VL"
+                           :myotherdomain.com "jL0OeCs9XOSsLUfCUKCRRSHKsQpM5WB3"))
+(verb-set-var "example.com" "IRMTk8T0RKgFxL3bL8KWv5FDDVtoe1VL")
+(verb-set-var "myotherdomain.com" "jL0OeCs9XOSsLUfCUKCRRSHKsQpM5WB3")
+```
+
+In above, a couple of advantages to using Elisp Prelude files is shown, such as having comments; dynamically getting email address from Emacs `user-mail-address` variable; and outputting a message, which can be useful for logging and debugging.
+
+For JSON environment file, the values (and even sub-values) are simply set using `verb-set-var` for each key value pair.  Example of JSON configuration (with not real keys or id's):
+
+``` json
+{
+  "email": "email@example.com",
+  "global_api_key": "hc33Vzco7TAkMKLNzIVgps7KiKLnUYIWJ7y9T",
+  "account_id": "Kb9SVZXtTVFYzt7rvgYv5WXJS31lh7OT",
+  "zone-ids":
+    {"example.com": "IRMTk8T0RKgFxL3bL8KWv5FDDVtoe1VL",
+     "myotherdomain.com": "jL0OeCs9XOSsLUfCUKCRRSHKsQpM5WB3"
+    }
+}
+```
+
+Results in this for `verb-show-vars` command:
+
+``` env
+email: email@example.com
+global_api_key: hc33Vzco7TAkMKLNzIVgps7KiKLnUYIWJ7y9T
+account_id: Kb9SVZXtTVFYzt7rvgYv5WXJS31lh7OT
+zone-ids: (:example.com IRMTk8T0RKgFxL3bL8KWv5FDDVtoe1VL :myotherdomain.com jL0OeCs9XOSsLUfCUKCRRSHKsQpM5WB3)
+example.com: IRMTk8T0RKgFxL3bL8KWv5FDDVtoe1VL
+myotherdomain.com: jL0OeCs9XOSsLUfCUKCRRSHKsQpM5WB3
+```
+
+This will result in the same `verb-show-vars` just above for the Elisp example (assuming `user-mail-address` is *"email@example.com"*). Notice, in the example, that `zone-ids` plist key value pairs are also flattened into their own key-value pairs for easy referencing.
```

Comment:
```suggestion
This will result in the same `verb-show-vars` just above for the Emacs Lisp example (assuming `user-mail-address` is *"email@example.com"*). Notice, in the example, that `zone-ids` plist key value pairs are also flattened into their own key-value pairs for easy referencing.
```